### ISSUE TITLE
Automate release publishing

### ISSUE THREAD

**Issue by bdupras** [bdupras/guava-probably#3]

**Comment by sriksun** [comment]
Any plans to release/publish this to central mvn repo soon ?

**Comment by bdupras** [comment]
Hi @sriksun I haven't had the need to keep this project maintained, but would be happy to take pull-requests to get it ready for automated testing & publishing. Or if you'd like to fork the code I can answer any questions you might have.

**Comment by nevillelyh** [comment]
Hi @bdupras we're also interested in using this lib. I can help with publishing if you already have a Sonatype project and can add me there (u:[neville](https://issues.sonatype.org/secure/ViewProfile.jspa?name=neville)) and as an editor in this repo.

**Comment by nevillelyh** [comment]
Published 1.0 today, using the following commands for posterity. I had to drop JDK 7 support since it's no longer supported in Travis.

```
export GPG_TTY=$(tty)
mvn -B release:prepare
mvn -B release:perform --settings=/path/to/release.xml # Sonatype username/password, etc.
```
